src/Worker/DelegatingWorkerPool.php

```diff
@@ -116,7 +121,7 @@ public function getWorker(): Worker
         return new PooledWorker($this->selectWorker(), $this->push(...));
     }
 
-    public function getLimit(): int
+    public function getWorkerLimit(): int
```

Comment:
Why did you rename this in-place? I guess it's still unreleased?

Reply:
Correct. No release has been made with `DelegatingWorkerPool` available.